### PR TITLE
CORE-3193 Fixed: Duplicated images on iOS 13

### PR DESCRIPTION
**Pull request by gonzalobarrios**
**What I did:**
Forked this repo
Fixed duplicated images on iOS 13

Before:
IMG_7844

Now: 
IMG_7866

### CODE REVIEW

**Comment by dentrubenkov** [comment]
@gonzalobarrios Looks like we have centered(should be aligned to bottom) image on the screenshot. 🤔

**Comment by gonzalobarrios** [comment]
> @gonzalobarrios Looks like we have centered(should be aligned to bottom) image on the screenshot. 🤔

**Comment by dentrubenkov** [comment]
@gonzalobarrios Should we do the similar changes in `didMoveToWindow` method.

File: UIImageViewAligned.swift [hunk 1]
```diff
@@ -202,10 +202,15 @@ open class UIImageViewAligned: UIImageView {
...
         super.didMoveToWindow()
         layer.contents = nil
-        if #available(iOS 11, *) {
+
+        if #available(iOS 13, *) {
+            self.layer.contents = CALayer()
+        } else if #available(iOS 11, *) {
             let currentImage = realImageView?.image
             image = nil
             realImageView?.image = currentImage
```
Comment (dentrubenkov) [inline UIImageViewAligned.swift]: Seems it should be applied for ios 13 too

**Pull request merged**